eng/pipelines/runtime.yml

```diff
@@ -64,8 +64,26 @@ jobs:
       - THIRD-PARTY-NOTICES.TXT
       - docs/*
       - src/installer/*
+      - src/mono/*
       - src/libraries/*
       - eng/pipelines/installer/*
+      - eng/pipelines/mono/*
+      - eng/pipelines/libraries/*
+    - subset: mono
+      include:
+      - src/libraries/System.Private.CoreLib/*
```

Comment:
Does mono depend on the native libraries? `src/libraries/Native`?

Reply:
for building mono? I don't think so. cc @akoeplinger

Reply:
The Mono build doesn't depend on `src/libraries/Native`.

Reply:
but what about the tests?

Reply:
We'll enable Mono runtime tests at a later point so that's not an issue for now.